publish-common/pom.xml

```diff
@@ -113,11 +113,17 @@
             <version>2.4.0</version>
             <scope>compile</scope>
         </dependency>
-        <!-- https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind --> 
+        <!-- https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind -->
         <dependency>
-           <groupId>com.fasterxml.jackson.core</groupId>
-           <artifactId>jackson-databind</artifactId>
-           </dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
```

Comment:
The Jackson databind version is coming from eiffel-remrem-parent. Should this also be added here?

Reply:
The Jackson-Core and Jackson-databind need to be the same version.
If you specify a specific version for jackson-databind in remrem-parent,, then you have to set same version to jackson-core in remrem-parent and have also the jackson-core depency here.

If these two libraries has different version at build time and not aligned, then it fails on the Jackson incopatible exception when executed.

So it either version is specified here or in remrem-parent, both libraries versions need to be aligned.